### ISSUE TITLE
Improve Parser Warning "Potential unsafe parsing"

### ISSUE THREAD

**Issue by LitschiW** [Cambio-Project/resilience-simulator#56]
In the parser Warning "[Warning] Potential unsafe parsing of value <name>" , `<name>` should be in quotation marks to make clear which value causes this warning.

**Comment by ImVis10** [comment]
@LitschiW I'd like to pick this up. Thank you:)

**Comment by LitschiW** [comment]
Sure go ahead :) , I'm currently slightly restructuring the repository. There should be a develop branch appearing soon which should be the target of all new PR (and the original fork will be detached). Just make sure your pull request targets the correct branch.

**Comment by ImVis10** [comment]
Sure, I'll make sure to point to the 'develop' branch once it's up.

**Comment by ImVis10** [comment]
@LitschiW, I don't think I have the permission to push to the push the changes to the upstream 'develop' branch.

**Comment by LitschiW** [comment]
Yes, both master and develop are protected and since you aren't a collaborator of the project you need to fork the repo, make your changes there and then create a pull request.

**Comment by ImVis10** [comment]
The **run_tests** step in the build pipeline failed for the [PR](https://github.com/Cambio-Project/resilience-simulator/pull/61)

**Comment by LitschiW** [comment]
The PR is fixed by now :) 
Please [check my comment](https://github.com/Cambio-Project/resilience-simulator/pull/61/files#r1010090474).
